src/components/application_manager/src/commands/mobile/delete_file_request.cc

```diff
@@ -87,24 +102,40 @@ void DeleteFileRequest::Run() {
   full_file_path += sync_file_name;
 
   if (file_system::FileExists(full_file_path)) {
+    LOG4CXX_DEBUG(logger_, "File exists! Path is " << full_file_path);
     if (file_system::DeleteFile(full_file_path)) {
+      LOG4CXX_DEBUG(logger_, "File deleted!");
       const AppFile* file = application->GetFile(full_file_path);
       if (file) {
         SendFileRemovedNotification(file);
       }
 
       application->DeleteFile(full_file_path);
       application->increment_delete_file_in_none_count();
-      SendResponse(true, mobile_apis::Result::SUCCESS);
+      response_params[strings::space_available] =
+          static_cast<uint32_t>(application->GetAvailableDiskSpace());
+      SendResponse(true,
+                   mobile_apis::Result::SUCCESS,
+                   "File successfully deleted",
+                   &response_params);
     } else {
-      SendResponse(false, mobile_apis::Result::GENERIC_ERROR);
+      LOG4CXX_DEBUG(logger_, "File not deleted!");
```

Comment:
@AByzhynar it looks like a trace logs

Reply:
@LuxoftAKutsan All these logs were added just for debugging purposes. Now they are deleted as redundant. 
Please check 5202f40d6d9bc3b62c6b6044e99fef849842edf3